edge-hub/README.md

```diff
@@ -13,18 +13,63 @@ This project contains the Edge Hub.
 
 
 ## How to debug Edge hub using Visual Studio
-1. Set environment variable `EdgeModuleHubServerCertificateFile` to the path of a SSL Certificate file (e.g. C:\edgeDevice.pfx); for debug purpose, you can [create](https://docs.microsoft.com/en-us/azure/cloud-services/cloud-services-certs-create#create-a-new-self-signed-certificate) and use a self-signed certificate.  Remember to restart Visual Studio to take effect.
-2. Update following values in appsettings_hub.json in Microsoft.Azure.Devices.Edge.Hub.Service project. 
-  * `IotHubConnectionString` - Edge Hub module connection string; set it as edge device connection string appending with "ModuleId=$edgeHub".
+1. Create an IOT edgeHub module connection string.
+Create a Visual studio project following process here: https://docs.microsoft.com/en-us/azure/iot-hub/iot-hub-portal-csharp-module-twin-getstarted Section Update the module twin using .NET device SDK
+but replace the program by the snippet below:
+```C#
+
```

Comment:
do you need this?
```suggestion
using Microsoft.Azure.Devices.Client;
using Microsoft.Azure.Devices.Shared;
```

Reply:
No I'll remove it.

Reply:
I need those: using System;
using Microsoft.Azure.Devices;
using Microsoft.Azure.Devices.Common.Exceptions;
using System.Threading.Tasks;

Reply:
you need to add "using Microsoft.Azure.Devices;" and "using Microsoft.Azure.Devices.Common.Exceptions;" in code snippet.

Reply:
I added it